src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -9938,62 +9938,59 @@ static void Main()
 
         [Fact]
         [WorkItem(54778, "https://github.com/dotnet/roslyn/issues/54778")]
-        public void GenericAttribute_Dynamic_01()
+        public void GenericAttribute_AttributeDependentTypes()
```

Comment:
Consider testing a nested situation (type is bad in multiple ways) to show that we only report once.
For example `List<string?>?` or `(int a, (int b, int c))`.